gtsam/discrete/AlgebraicDecisionTree.h

```diff
@@ -89,24 +89,24 @@ namespace gtsam {
     }
 
     /** Create from keys and vector table */
-    AlgebraicDecisionTree //
-    (const std::vector<typename Base::LabelC>& labelCs, const std::vector<double>& ys) {
-      this->root_ = Base::create(labelCs.begin(), labelCs.end(), ys.begin(),
-          ys.end());
+    AlgebraicDecisionTree(const typename Base::LabelCs& labelCs,
```

Comment:
This is wrong! The labelCs here and the vectors have to correspond: a map will sort and will break this.